### ISSUE TITLE
[Enhancement] Date/Time display options

### ISSUE THREAD

**Issue by MikeTheTux** [LycheeOrg/Lychee#1211]
I would like to request a config option for the representation of date/time values, e.g. in the "Photo overlay" when using "Date taken" as:

Current default:
`Jan 7, 2022, 9:38:44 PM UTC+01:00`

I wish at least the option to select 24 hour format for the time and the "European" format for the date:
`7 Jan 2022, 21:38:44 UTC+01:00`

**Comment by nagmat84** [comment]
Note that the format is automatically chosen by the JS engine of your browser and follows the locale setting of your browser and/or your operating system. For example, on a German Windows with a German Firefox the format is exactly as you propose.

Nonetheless, I agree that we could add an option which allows you to use the the browser's default or a specific setting.

**Comment by MikeTheTux** [comment]
> Note that the format is automatically chosen by the JS engine of your browser and follows the locale setting of your browser and/or your operating system. For example, on a German Windows with a German Firefox the format is exactly as you propose.

You are right. I reported the finding using English Windows in combination with English Firefox.

On my German Windows with German Firefox I receive a localized string as expected:

`07.01.2022, 21:38:44 UTC+01:00`.

My German Android mobile, using the default Samsung Webbrowser is showing also the English/American variant. This, in combination with the English Windows system from above, brought me on the wrong track ...
Firefox and Chrome on the same Android device is again using the European, localized string as expected.

In summary: It's not a high prio topic, but having an option to override the browser's default would be a nice improvement.
